core/src/com/unciv/ui/audio/MusicController.kt

```diff
@@ -209,7 +209,17 @@ class MusicController {
             ControllerState.Silence ->
                 if (++ticksOfSilence > silenceLengthInTicks) {
                     ticksOfSilence = 0
-                    chooseTrack()
+                    
+					val worldScreen = UncivGame.Current.getWorldScreenIfActive()
+                    if (worldScreen == null ||
+                            !chooseTrack(
+                                prefix = MusicMood.peaceOrWar(!worldScreen.viewingCiv.isAtWar()), 
+                                suffix = MusicMood.Ambient, 
+                                EnumSet.of(MusicTrackChooserFlags.NoMatchingPrefix, MusicTrackChooserFlags.SuffixMustMatch)
+                            )
+                    ) {
+                        chooseTrack()
+                    }
```

Comment:
This is a strange way of doing this. I would assume that chooseTrack() internally would, if no matching tracks exist for the given filters, try and choose a track with no filters